letsencrypt/build.yaml

```diff
@@ -8,23 +8,3 @@ build_from:
 codenotary:
   signer: notary@home-assistant.io
   base_image: notary@home-assistant.io
-args:
```

Comment:
Passing the version numbers through this metadata is a feature really. If you build using `docker build` manually just use `--build-arg` for each one of these.

Since we use them in all add-ons we should keep them here too.

Reply:
My iteration method was as described in the [Local aad-on testing document](https://developers.home-assistant.io/docs/add-ons/testing). 
* I checked out the addons repo
* VSCode + devcontainer
* Tasks: Start Homeassistant
* I would manually make changes to the letsencrypt addon
* Comment out the image from `config.yaml` (during iteration)
* Reinstall addon in the bespoke homeassistant

During this iteration process, the args would not be injested, nor would pip be available to the built-in build process (which I haven't gotten into). 

Please let me know if there is anything else I need to be aware of during the iteration process

Reply:
I will trustfall to @agners and undo pipeline changes